Common/Transforms/itkCyclicBSplineDeformableTransform.hxx

```diff
@@ -42,7 +42,7 @@ CyclicBSplineDeformableTransform<TScalarType, NDimensions, VSplineOrder>::SetGri
   /** Check if last dimension of supportregion < last dimension of grid. */
   const int lastDim = this->m_GridRegion.GetImageDimension() - 1;
   const int lastDimSize = this->m_GridRegion.GetSize(lastDim);
-  const int supportLastDimSize = this->m_SupportSize.GetElement(lastDim);
+  const int supportLastDimSize = VSplineOrder + 1;
```

Comment:
@mstaring @stefanklein The original code specifically picked the last dimension from the support size, but as far as I can see, all of its dimensions are the same: `SplineOrder + 1`.